core/controllers/admin.py

```diff
@@ -1309,13 +1310,20 @@ def get(self) -> None:
             user_settings = user_services.get_user_settings(user_id)
             user_roles = user_settings.roles
             managed_topic_ids = []
+            coordinated_language_ids = []
             if feconf.ROLE_ID_TOPIC_MANAGER in user_roles:
                 managed_topic_ids = [
                     rights.id for rights in
                     topic_fetchers.get_topic_rights_with_user(user_id)]
+            if feconf.ROLE_ID_TRANSLATION_COORDINATOR in user_roles:
+                coordinated_language_ids = [
+                    rights.language_id for rights in
+                    suggestion_services.get_translation_rights_with_user(
```

Comment:
`get_translation_rights_with_user` function inside `suggestion_services` doesn't sound right to me as the suggestion should not be serving user's permissions/rights. Move it to  rights_services.

Reply:
I took a quick pass an seems like all the permission models/object/services are inside the suggestion context, can we move it to user or rights context?

Reply:
Since the rights were related to suggestions so I added it in the `suggestion_services`. Moved it to `user_services`. PTAL.